sdk/lib/core/list.dart

```diff
@@ -674,6 +674,22 @@ abstract class List<E> implements EfficientLengthIterable<E> {
    * A range from [start] to [end] is valid if `0 <= start <= end <= len`, where
    * `len` is this list's `length`. The range starts at `start` and has length
    * `end - start`. An empty range (with `end == start`) is valid.
+   *
+   *     List<int> list1 = new List(3);
```

Comment:
I'd start with:
```
 * 
 * Example:
```
to set the expectation.

Also, change `list1` to just `list`, the "1" doesn't add anything to the example.